docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"resolved"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
+```
+Subscribes a handler function for an event from promise lifecycle ("created", "chained", "fulfilled", "rejected", "resolved" and "cancelled").
```

Comment:
We need to explain when each event is fired - even in one line. We also need to emphasize whether or not events happen across bluebird instances.